styles.css

```diff
@@ -0,0 +1,88 @@
+.content {
+  display: flex;
+  flex-direction: column;
+}
+
+@media (max-width: 500px) {
+  body {
+    background-image: url("media/header-shapes mobile.png");
```

Comment:
- **[OPTIONAL]** Great job on applying the background image through CSS. 🥇  However, your background image is repeating which does not match the Figma template. It would be nice to fix the background image as it appeals to your design. 💯 😃 
 
<img width="376" alt="Screenshot 2023-01-19 at 4 38 34 PM" src="https://user-images.githubusercontent.com/24832846/213427144-99a4c517-6707-4182-8406-27554d7df45b.png">

Figma Design
<img width="350" alt="Screenshot 2023-01-19 at 4 39 50 PM" src="https://user-images.githubusercontent.com/24832846/213427410-f3809e50-70b6-419c-9a59-f065c47b6259.png">